tests/test_live.py

```diff
@@ -220,3 +220,33 @@ def test_large_request(platform):
         E2_keV=200.0,
         scw_list=[f"0665{i:04d}0010.001" for i in range(200)],
     )
+
+@pytest.mark.slow
+#@pytest.mark.parametrize("platform", ["staging-1-3", "staging-1-2", "production-1-2"])
+#def test_pickling(platform):
+def test_pickling():
```

Comment:
What is this test actually testing?

Reply:
very peculiar pickling problem! Renamed.